test/e2e/common/downwardapi_volume.go

```diff
@@ -21,7 +21,6 @@ import (
 	"time"
 
 	"k8s.io/api/core/v1"
```

Comment:
Let's rename this file to `downward_api_volume.go` for consistency. Or rename the new file to `downwardapi.go`.